core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -917,17 +906,6 @@ protected void postImportOSM() {
             maxSpeedCalculator.close();
         }
 
-        // Important note: To deal with via-way turn restrictions we introduce artificial edges in OSMReader (#2689).
-        // These are simply copies of real edges. Any further modifications of the graph edges must take care of keeping
```

Comment:
This comment is still true and not only related to graph sorting so I would keep it.

Reply:
Maybe at the beginning of postImportOSM? It does not really matter. If you want we can also delete it and just keep it in mind :)